peachjam/tasks.py

```diff
@@ -28,6 +28,29 @@ def update_document(ingestor_id, document_id):
     log.info("Ingestor is disabled, ignoring.")
 
 
+@background(queue="peachjam", remove_existing_tasks=True)
```

Comment:
Put a delay on this task of a few minutes (see comment above about delayed deletion).

Reply:
Set the delay to 5 minutes.